llabake/Hello-Books
# Ft delete and modify review 154746870

#### What does this PR do?
This PR implements get all books using react and redux

#### Description of Task to be completed?
User should be able to edit and delete a review 

#### How should this be manually tested?
* clone the repo
* cd into the project directory
* Run npm install to install all the dependency
* Run npm run dev
* visit the port where the webpack-dev-server started
* on successful login, and adding a review the icons to delete or edit on clicked should the do the respective action
#### What are the relevant pivotal tracker stories?
154746870

#### Any background context you want to add?
Run npm run start-dev
#### Screenshots
![image](https://user-images.githubusercontent.com/29184825/35927281-9eac5382-0c2a-11e8-9182-67aaf8aa8b36.png)
